CMakeLists.txt

```diff
@@ -33,6 +33,10 @@ if(RCUTILS_NO_FILESYSTEM)
   add_compile_options(-DRCUTILS_NO_FILESYSTEM=1)
 endif()
 
+if(RCUTILS_NO_THREAD_SUPPORT)
```

Comment:
From my point of view, it is more elegant to add this compile options as in https://github.com/micro-ROS/rcutils/blob/8404c5dfc0ef47889a4eb9f90b6fc4d0a9116db1/CMakeLists.txt#L120

Reply:
Done